charts/azuremonitor-containers/values.yaml

```diff
@@ -21,8 +21,8 @@ Azure:
 omsagent:
   image:
     repo: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod"
-    tag: "ciprod01312022"
-    tagWindows: "win-ciprod01312022"
+    tag: "ciprod03112022"
```

Comment:
@sarahpeiffer , you will have to bump chart version from 2.9.1 to 2.9.2 in chart.yaml file